integration-cli/docker_cli_ps_test.go

```diff
@@ -436,6 +436,10 @@ func (s *DockerSuite) TestPsListContainersFilterLabel(c *testing.T) {
 }
 
 func (s *DockerSuite) TestPsListContainersFilterExited(c *testing.T) {
+	// TODO Flaky on  Windows CI [both RS1 and RS5]
+	// On slower machines the container may not have exited
+	// yet when we filter below by exit status/exit value.
+	testRequires(c, DaemonIsLinux)
```

Comment:
Can we use `skip.If()` for this, and put a descriptive comment in there, so that it's more visible why we're skipping, and that it's a TODO; similar to https://github.com/moby/moby/blob/32688a47f34181af671157eff9bc9c5697b321bd/pkg/chrootarchive/archive_test.go#L176

Perhaps put the number of the tracking issue in there, something like;

```suggestion
	skip.If(c, DaemonIsWindows(), "FLAKY on Windows, see #20819")
```

Reply:
@thaJeztah - thanks for suggesting this change. I had noticed that this pattern was not used anywhere in the integration-cli tests. So I had decided against using it. However it is very commonly used in the integration tests. 

I am going to move ahead with your suggestion and we shall figure out soon if there is some reason why the integration-cli tests avoid this even though it is much more informative than just skipping it.